scootapi/setup/setup.go

```diff
@@ -0,0 +1,57 @@
+package setup
+
+import (
+	"fmt"
+
+	"github.com/scootdev/scoot/scootapi"
+)
+
+// Main is the entry point for Cloud Scoot setup.
+// cmds is running commands
+// strategies maps strings to SchedulerStrategy
+// which is the name of the strategy to use
+// args is the remaining arguments
+func Main(cmds *Cmds, strategies map[string]SchedulerStrategy, which string, args []string) error {
```

Comment:
nit: I dont know what which means can we give this a name that is more descriptive. I think this is just the user input strategy.